src/tracks/playabletrack/notetrack/ui/NoteTrackVZoomHandle.cpp

```diff
@@ -72,17 +70,12 @@ HitTestPreview NoteTrackVZoomHandle::HitPreview(const wxMouseState &state)
       ::MakeCursor(wxCURSOR_MAGNIFIER, ZoomOutCursorXpm, 19, 15);
    static  wxCursor arrowCursor{ wxCURSOR_ARROW };
 
-   bool bVZoom;
-   gPrefs->Read(wxT("/GUI/VerticalZooming"), &bVZoom, false);
-   bVZoom &= !state.RightIsDown();
-   const auto message = bVZoom ? 
-      XO("Click to vertically zoom in. Shift-click to zoom out. Drag to specify a zoom region.") :
-      XO("Right-click for menu.");
+   const auto message = 
+      XO("Click to vertically zoom in. Shift-click to zoom out. Drag to specify a zoom region.");
```

Comment:
This change is not quite equivalent to the previous version.

"Right-click for menu." was a message that appeared in the status bar, when you right click, and have not yet released the button.  This is lost.

But I suppose you observed this, and the answer is that it doesn't make sense as a discoverability aid if you really have to do that right click before you see the message suggesting right click!

And you are just assuming right click for context menu is a widespread convention we don't need to mention.

Yes?

Reply:
> And you are just assuming right click for context menu is a widespread convention we don't need to mention.

@Paul-Licameli  that is a _**very**_ widespread convention an many apps and OSs

Reply:
indeed